src/events/funStuff.js

```diff
@@ -1,48 +1,20 @@
-const { Events, PermissionsBitField } = require("discord.js");
-const fs = require("fs");
-const path = require("path");
-const util = require('util');
-const exec = util.promisify(require('child_process').exec);
-
-// Path to the reactionMap.json file
-const reactionMapPath = path.join(__dirname, "..", "reactionMap.json");
-let reactionMap = loadReactionMap();
-
-function loadReactionMap() {
-    try {
-        const rawData = fs.readFileSync(reactionMapPath, "utf8");
-        return JSON.parse(rawData);
-    } catch (error) {
-        console.error("Error loading reactionMap.json", error);
-        return {};
-    }
-}
-
-async function gitCommit(key, value) {
-    try {
-        await exec(`git add ${reactionMapPath}`);
-
-        await exec(`git commit -m "Update reactionMap.json: ${key}: ${value}"`);
-
-        await exec("git push origin main");
-        console.log(`Pushed changes to remote: ${key}: ${value}`);
-    } catch (error) {
-        console.error(`exec error: ${error}`);
-        return;
-    }
-}
+const { Events } = require("discord.js");
+const { loadReactionMap } = require("../utils/reactionutils");
 
 module.exports = {
     name: Events.MessageCreate,
     async execute(message) {
         if (message.author.bot) return;
-
+        const reactionMap = loadReactionMap();
         for (const [keyword, reaction] of Object.entries(reactionMap)) {
             const regex = new RegExp(
                 `(?:^|[^a-zA-Z0-9])${keyword}(?:$|[^a-zA-Z0-9])`,
                 "i"
             );
-            if (Math.random() < Math.random()*Math.random() && regex.test(message.content)) {
+            if (
+                Math.random() < Math.random() * Math.random() &&
```

Comment:
I think we should reduce this to either .25 (what it is now) or .33 to make it a bit easier to understand what the chances are here